DataStructures/HashMap/Hashing/HashMap.java

```diff
@@ -127,20 +201,36 @@ public boolean isEmpty() {
     public static class Node {
     	private Node next;
     	private int key;
-    	
+			
+			/**
+			 * Constructor to create a new node and assign the initial key
+			 * @param key the key of a node 
+			 */
     	public Node(int key) {
     		next = null;
     		this.key = key;
     	}
-    	
+			
+			/**
+			 * Get the next node after the current node
+			 * @return Node next node
+			 */
```

Comment:
Fix indentation example

Reply:
Before this gets merged, I just re-checked it and found some indentation errors, can you fix these?

Reply:
@afadhitya